### ISSUE TITLE
What properties are added to Rex?

### ISSUE THREAD

**Issue by Igor-Palaguta** [neilpa/Rex#145]
How do you define UI properties that are in Rex. 

E.g. Can we expect UIButton rex_image property or UIPageControl rex_currentPage? Or very useful NSLayoutConstraint rex_constant and rex_priority properties?

Sure UIKit has lots of different properties and adding them all will make this simple tiny library very complex. But I think there are still lots of often used that can be also added

Thanks!

**Comment by andersio** [comment]
You can always suggest new properties. :) I also foresee the dramatic(?) growth in RAC5 after deploying gyb (and introducing the `BindingConsumer` protocol, perhaps).

**Comment by RuiAAPeres** [comment]
@Igor-Palaguta I agree with @andersio, if there is something you would like to see, please let us know.
